plugins/MauticFocusBundle/EventListener/ReportSubscriber.php

```diff
@@ -108,17 +187,123 @@ public function onReportBuilder(ReportBuilderEvent $event): void
             'display_name' => 'mautic.focus.graph.stats',
             'columns'      => $columns,
         ];
-        $context = self::CONTEXT_FOCUS_STATS;
 
-        $event->addTable($context, $data, self::FOCUS_GROUP);
+        $event->addTable(self::CONTEXT_FOCUS_STATS, $data, self::FOCUS_GROUP);
+        $this->addFocusLeadsTable($event, $columns);
```

Comment:
This should probably be evaluated only for lead context:
```
if (!$event->checkContext([self::CONTEXT_FOCUS_LEADS])) {
    $this->addFocusLeadsTable($event, $columns);
}
```

Reply:
ok, I changed this.